### ISSUE TITLE
Ink Color

### ISSUE THREAD

**Issue by mtg1986** [IBVSoftware/CollaBoard-Feedback#595]
If I work with the two pens on the Surface Hub, both start with the blue color. Then I change the left pen to red. The left one now inks red, the right one still inks blue. But if I want to change the color of the right pen now, the color menu shows that it is red, because the left one is red already. If I change the right one to green now, it inks green, the left inks red. Changing the color of the left one again, the menu shows green, because I selected green with the right one before.
So, basically, the color menu doesn't differ the pens anymore, the actual inking works correctly though.

Tested on Surface Hub, Version 2.0.24

**Comment by gsantopaolo** [comment]
@mtg1986 this is by design. If you look at what whiteboard does is pretty similar....
